data/json/regional_map_settings.json

```
@@ -135,7 +137,8 @@
           "f_lily": 1,
           "f_flower_tulip": 1,
           "f_mutpoppy": 1
-        }
+        },
+        "f_region_water_plant": { "f_cattail": 15, "f_lilypad": 1, "f_lotus": 5 }
```

It is `f_cattails` - see https://github.com/CleverRaven/Cataclysm-DDA/blob/d7fe8d780640f2820d6b0a12075e64d247f114e6/data/json/furniture_and_terrain/furniture-flora.json#L87.

```suggestion
        "f_region_water_plant": { "f_cattails": 15, "f_lilypad": 1, "f_lotus": 5 }
```